doc/Language/objects.pod6

```diff
@@ -526,8 +523,7 @@ class MyClass {
 }
 =end code
 
-If all you need to do is checking things after the actual construction,
-or modify attributes after the object construction, writing a submethod
+If after object construction, you need to check things or modify attributes, writing a submethod
```

Comment:
How about:

`The C<TWEAK> method allows you to check things or modify attributes after object construction:`

from 

`If after object construction, you need to check things or modify attributes, writing a submethod
C<TWEAK> is a good approach (available in v6.d / Rakudo 2016.11):`